checkfiles.py

```diff
@@ -1016,10 +1016,11 @@ def fetch_files(session, url, search_query, out, include_unexpired_upload=False,
             upload_credentials = r.json()['@graph'][0]['upload_credentials']
             try: 
                 if fileObject.json()['s3_uri']:
-                    job['s3_uri'] = fileObject.json()['s3_uri']
+                    job['download_url'] = fileObject.json()['s3_uri']
             except KeyError:
-                errors['s3_uri_missing'] = ('s3 uri is not present for this file')
-
+                job['download_url'] = upload_credentials['upload_url']
+                if not job['download_url']:
+                    errors['download_url_missing'] = ('download url is missing')
```

Comment:
I am wondering what will happen if upload_credentials doesn't have "upload_url" ? would not it throw Exception here? Should not it be another try/catch there?

> try:
    job['download_url'] = fileObject.json()['s3_uri']
except:
    try:
        job['download_url'] = upload_credentials['upload_url']
    except:
        errors['download_url_missing'] = ('download url is missing')